### DISCUSSION TITLE
Write down radiator consumption and the total for the house by hand - Anotar a mano consumo de radiador y el total de la vivienda

### DISCUSSION THREAD

**Discussion by afternet** [jmcollin78/versatile_thermostat#553] [General]
Hello everyone!
I have been using this very useful integration for a year and now I need a function that I think existed before, or perhaps I configured it when I registered the radiators. In any case, now it does not let me edit it and I think it is important.
I have 5 electric radiators registered as a switch. They do not have an energy meter, but I do know how much each one of them consumes.
So I need to manually enter the consumption in watts, in addition to the total consumption of the house, which is 3450w, so that the integration can disconnect them if necessary so that the main switch does not trip.
In the second image you can see 5 heaters: Bathroom 700w, Office 1000w, Gabriel's Bedroom 1000w, Master Bedroom 2000w and Living Room 2000w. Knowing that the total consumption of the house is 3450w and with a Shelly EM meter measuring it constantly, can't the heaters be turned on or off alternately depending on the total consumption of the house?
I perfectly remember that this function existed, and now it does not let me edit it. It is something important that if it causes the main switch of the house to trip, then the integration would not work for me.
I am sending a screenshot of what it now allows you to do, and that is to choose the radiator consumption sensor, which in my case does not exist and I need to write down the consumption that it has by hand.
Thank you for this wonderful integration and I hope you see this improvement as important as I do.

_________________________________________________________________

Buenas a todos!
Llevo un año utilizando esta integración muy útil y ahora necesito una función que creo que antes si existía, o quizás la configuré al dar de alta los radiadores. En cualquier caso ahora no me deja editarla y creo que es importante.
Tengo 5 radiadores eléctricos dados de alta como interruptor. Carecen de medidor de energía, pero yo si sé cuanto consumen cada uno de ellos.
Entonces necesito poner a mano el consumo en watios, además del consumo total de la vivienda que son 3450w, para que la integración vaya desconectandolos si fuera necesario para que no salte el interruptor general.
En la segunda imágen se ven 5 calefacciones: Baño 700w, Despacho 1000w, Dorm.Gabriel 1000w, Dorm.Principal 2000w y Salón 2000w. Sabiendo que el consumo total de la vivienda son 3450w y con un medidor Shelly EM midiéndolo constantemente, no pueden ir encendiéndose o apagándose los calefactores encendidos alternativamente según el consumo total de la vivienda?
Recuerdo perfectamente que esa función existía, y ahora no me deja retocarla. Es algo importante que si provoca el salto del interruptor general de la vivienda pues no me serviría la integración.
Mando pantallazo de lo que ahora permite hacer, y es elegir el sensor de consumo del radiador, que en mi caso no existe y necesito anotar el consumo que tiene a mano.
Gracias por esta maravillosa integración y ojalá veas esta mejora tan importante como la veo yo.

![image](https://github.com/user-attachments/assets/b3c093ba-317a-49c1-a8e4-5e0a077c145c)

![image](https://github.com/user-attachments/assets/c749800c-7962-4978-92da-57cf16cb2c6e)

**Comment by jmcollin78** [comment]
> In the second image you can see 5 heaters: Bathroom 700w, Office 1000w, Gabriel's Bedroom 1000w, Master Bedroom 2000w and Living Room 2000w. Knowing that the total consumption of the house is 3450w and with a Shelly EM meter measuring it constantly, can't the heaters be turned on or off alternately depending on the total consumption of the house?

Yes ! This is the power management function of Versatile Thermostat. In the first screen copy, you have to insert your Shelly power entity which is measuring the power consumption of your house. 1000w is not an entity_id. That is what the error message says.

In the second field named "max power", you have to insert an entity_id of a entity that gives the max power you want. This entity can surely be a Helper of type input_number. You can then change the value of this input_number to have a dynamic max power in your house. I hope this helps.

**Comment by afternet** [comment]
Hello again!!! I have a question, sorry.

In "POWER SENSOR ENTITY ID" I have put the input_number that I have created of 1000w.
In "MAX POWER SENSOR ENTITY ID" I have put another input_number of 3450w which is the total of the house.
But I don't see where to configure the Shelly EM sensor that is measuring the real-time consumption of the house.

If I don't provide that information, I don't understand how the integration will be able to automatically activate and deactivate the heating depending on the consumption of the house.

Thanks for everything.

![Screenshot from 2024-10-16 17-55-33](https://github.com/user-attachments/assets/8c5cd166-18af-4cd7-a686-4f4beeb80457)
![Screenshot from 2024-10-16 17-57-48](https://github.com/user-attachments/assets/b5b337e3-f1c3-4153-badf-4aa80a3854c0)

________________________________________________________________

Buenas de nuevo!!! Tengo una duda, perdonar.

En "POWER SENSOR ENTITY ID" he puesto el input_number que he creado de 1000w.
En "MAX POWER SENSOR ENTITY ID" he puesto otro input_number de 3450w que es el total de la vivienda.
Pero no acabo de ver donde configurar el sensor Shelly EM que me está midiendo el consumo a tiempo real de la vivienda.

Si ese dato no lo aporto no acabo de entender como va a poder la integración activar y desactivar automáticamente las calefacciones dependiendo del consumo de la vivienda.

Gracias por todo.

**Reply by jmcollin78** [reply]
Hello @afternet,

> In "POWER SENSOR ENTITY ID" I have put the input_number that I have created of 1000w.
> In "MAX POWER SENSOR ENTITY ID" I have put another input_number of 3450w which is the total of the house.

This is not good. In the "POWER SENSOR ENTITY ID" you should set the power measurement of your house.
In the "MAX POWER SENSOR ENTITY ID" you should set a input_number which hold the max power you want for house, so that, if this power will be exceeded by turning on a heater, the heater will NOT be turned on.

**Comment by afternet** [comment]
I thought that somewhere I had to tell them exactly how much power each radiator has, but I seem to remember that I already configured that when I added the integration a long time ago.
It is showing me on screen the consumption of each radiator without them having a power meter, it must be because I installed it.
Well, it seems that it works.
Thanks a lot


Pensaba que en algún sitio había que decirle que potencia tiene exactamente cada radiador, pero creo recordar que eso ya lo configuré cuando añadí la integración hace bastante tiempo.
Me está mostrando en pantalla el consumo de cada radiador sin tener ellos un medidor de potencia, debe ser porque se lo puse yo.
Bueno, parece que si funciona.
Mil gracias
